qxf2_scheduler/base_gcal.py

```diff
@@ -105,4 +111,46 @@ def make_day_busy(fetch_date):
     end_date = process_date_isoformat(end_date)
     busy_slots=[{'start':start_date,'end':end_date}]
 
-    return busy_slots    
\ No newline at end of file
+    return busy_slots
+
+
+def create_event_for_fetched_date_and_time(service,email,create_event_start_time,create_event_end_time):
```

Comment:
Can you make this method a little more configurable? Right now you are hard-coding the description and attendees. Instead, hard code those values in qxf2_scheduler and then pass them as arguments to this method. This way, when we enhance the application to add a proper message and pass interviewers, etc. we will not need to change base_gcal.py. That will help.

Also, can you change the argument names from `create_event_start_time`, `create_event_end_time` to `start_time`, `end_time`

Reply:
Done